layouts/index.html

```diff
@@ -494,29 +494,40 @@ <h4 class="fw-bold mb-0 pb-2">{{ i18n "Effortless to switch to" }}</h4>
-                                <img loading="lazy" src="/brands/AlmaCarePoweredByTuxCare.svg" alt="AlmaCare" class="al-commercial-supporter-logo">
-                            </div>
-                        </a>-->
-   			 <!-- OpenLogic -->
-                        <a href="https://www.openlogic.com/">
-                            <div class="col d-flex flex-fill align-items-center justify-content-center al-commercial-supporter-logo-container">
-                                <img loading="lazy" src="/brands/logo-openlogic-tagline-white.svg" alt="OpenLogic" class="al-commercial-supporter-logo">
-                            </div>
-                        </a>
-             <!-- Cybertrust Japan -->
-                        <a href="https://www.cybertrust.co.jp/">
-                            <div class="col d-flex flex-fill align-items-center justify-content-center al-commercial-supporter-logo-container">
-                                <img loading="lazy" src="/brands/cybertrust-japan.svg" alt="Cybertrust Japan" class="al-commercial-supporter-logo">
-                            </div>
-                        </a>
+				<p class="text-center pb-2 mb-5 border-bottom">Minimize your risk while getting the most out of your AlmaLinux systems</p>
+				
+				<!-- OpenLogic row -->
+                <div class="row al-commercial-supporter-list">
+				
+                    <div class="col-12 col-md-4 al-commercial-supporter-logo-container d-flex flex-fill justify-content-center">
+					    <a href="https://www.openlogic.com/">
+                            <img loading="lazy" src="/brands/logo-openlogic-tagline-white.svg" alt="OpenLogic" class="al-commercial-supporter-logo">
+						</a>
+                     </div>
+					 
+					 <div class="col-12 col-md-8">
+						<p>Confidently deploy AlmaLinux and all your open source packages with enterprise-grade, 24/7/365 support backed by guaranteed SLAs. Keep your mission-critical apps running smoothly with unlimited tickets and direct access to experienced enterprise architects who put your business first.</p>
+						<a href="https://tuxcare.com/almalinux-commercial-support/?utm_source=website&utm_medium=banner&utm_campaign=homepage">Get support</a>
+                     </div>
+                 </div>
+				 
+				 <!-- CyberTrust row -->
+                 <div class="row al-commercial-supporter-list">
+                     <div class="col-12 col-md-4 order-md-last al-commercial-supporter-logo-container d-flex flex-fill justify-content-center">
+					     <a href="https://www.cybertrust.co.jp/">
+                             <img loading="lazy" src="/brands/cybertrust-japan.svg" alt="Cybertrust Japan" class="al-commercial-supporter-logo">
+						 </a>
+                     </div>
+					 
+					<div class="col-12 col-md-8">
+					    <p>AlmaCare is an enterprise-grade AlmaLinux support service delivered by one of the founding sponsors of the AlmaLinux Project, TuxCare, a division of CloudLinux Inc.</p>
```

Comment:
```suggestion
					    <p>Cybertrust is a provider of a comprehensive security certification services to various public and private entities globally, and offers comprehensive support for AlmaLinux users. Have also produced MiracleLinux for nearly 20 years.</p>
```

I think this is getting confused because we took TuxCare off the site, but adjusting for CyberTrust.